### ISSUE TITLE
deploy_conversion_hosts playbook doesn't work

### ISSUE THREAD

**Issue by sigtryggr** [os-migrate/os-migrate#483]
playbook is stopping at 'deploy the src conversion host inventory' task:

TASK [deploy the src conversion host inventory] *******************************************************************************************************************************************************************
task path: /root/.ansible/collections/ansible_collections/os_migrate/os_migrate/playbooks/deploy_conversion_hosts.yml:4
<localhost> ESTABLISH LOCAL CONNECTION FOR USER: root
<localhost> EXEC /bin/sh -c 'echo ~root && sleep 0'
<localhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /root/.ansible/tmp/ansible-tmp-1631215420.1711073-154250634847659 `" && echo ansible-tmp-1631215420.1711073-154250634847659="` echo /root/.ansible/tmp/ansible-tmp-1631215420.1711073-154250634847659 `" ) && sleep 0'
Using module file /usr/lib/python3/dist-packages/ansible/modules/utilities/logic/include_role.py
<localhost> PUT /root/.ansible/tmp/ansible-local-414550qfvnz8oc/tmpmpmg5yt3 TO /root/.ansible/tmp/ansible-tmp-1631215420.1711073-154250634847659/AnsiballZ_include_role.py
<localhost> PUT /root/.ansible/tmp/ansible-local-414550qfvnz8oc/tmp6bf3r5fh TO /root/.ansible/tmp/ansible-tmp-1631215420.1711073-154250634847659/args
<localhost> EXEC /bin/sh -c 'chmod u+x /root/.ansible/tmp/ansible-tmp-1631215420.1711073-154250634847659/ /root/.ansible/tmp/ansible-tmp-1631215420.1711073-154250634847659/AnsiballZ_include_role.py /root/.ansible/tmp/ansible-tmp-1631215420.1711073-154250634847659/args && sleep 0'
<localhost> EXEC /bin/sh -c '/usr/bin/python3 /root/.ansible/tmp/ansible-tmp-1631215420.1711073-154250634847659/AnsiballZ_include_role.py /root/.ansible/tmp/ansible-tmp-1631215420.1711073-154250634847659/args && sleep 0'
<localhost> EXEC /bin/sh -c 'rm -f -r /root/.ansible/tmp/ansible-tmp-1631215420.1711073-154250634847659/ > /dev/null 2>&1 && sleep 0'
fatal: [localhost]: FAILED! => {
    "changed": false,
    "module_stderr": "",
    "module_stdout": "",
    "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",
    "rc": 0
}

What can I do to understand the exact cause of the error?

**Comment by jistr** [comment]
Looks like an error in the Ansible's built in `include_role` module, i've never seen this before. I'm suspecting some sort of version/compatibility issue. Can you post the output of `ansible --version` ?

**Comment by sigtryggr** [comment]
> Looks like an error in the Ansible's built in `include_role` module, i've never seen this before. I'm suspecting some sort of version/compatibility issue. Can you post the output of `ansible --version` ?

ansible 2.9.6
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/rootadmin/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.8.10 (default, Jun  2 2021, 10:49:15) [GCC 9.4.0]

**Comment by jistr** [comment]
Interesting, we test with:

```
ansible 2.9.25
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /root/venv/lib64/python3.8/site-packages/ansible
  executable location = /root/venv/bin/ansible
  python version = 3.8.11 (default, Jun 29 2021, 00:00:00) [GCC 11.1.1 20210531 (Red Hat 11.1.1-3)]
```

So your versions (Ansible 2.9 + Python 3.8) look good at first glance. I'm wondering whether there's  been some bug fix in the later patch versions of Ansible 2.9. 

-----

Can you try running with latest patched Ansible 2.9, probably using a virtualenv as described in the docs? Here:
https://os-migrate.github.io/os-migrate/user/install-from-galaxy.html#using-virtualenv 

You can then run `ansible --version` again to verify that you are using virtualenv's Ansible and not the system installation before retrying.
